mobile-app/app/screens/AppNavigator/screens/Dex/helpers/path-finding.tsx

```diff
@@ -0,0 +1,95 @@
+
+export interface GraphProps { pairId: string, a: string, b: string }
+/**
+ * To find all the adjacent vertices / neighbors of a node in a graph
+ * @param startNode
+ * @param graph
+ */
+export function getAdjacentVertices (startNode: string, graph: GraphProps[]): string[] {
+  return (
+    graph.reduce((vertices: string[], vertex): string[] => {
```

Comment:
Is map a better option here instead of reduce?

Reply:
Also, might consider just pushing it on a new array instead of recreating the array everytime.